dll/win32/netid/lang/ro-RO.rc

```diff
@@ -2,93 +2,93 @@
  * FILE:          dll/win32/netid/lang/ro-RO.rc
  *                ReactOS Project (https://reactos.org)
  * TRANSLATORS:   Copyright 2011-2018 Ștefan Fulea <stefan.fulea@mail.com>
- *                Copyright 2023 Andrei Miloiu <miloiuandrei@gmail.com>
+ *                Copyright 2023-2024 Andrei Miloiu <miloiuandrei@gmail.com>
  */
 
 LANGUAGE LANG_ROMANIAN, SUBLANG_NEUTRAL
 
 IDD_PROPPAGECOMPUTER DIALOGEX 0, 0, 256, 218
 STYLE DS_SHELLFONT | WS_CHILD | WS_DISABLED | WS_CAPTION
-CAPTION "Numele calculatorului"
+CAPTION "Nume computer"
 FONT 8, "MS Shell Dlg"
 BEGIN
     ICON IDI_COMPNAME, IDC_STATIC, 4, 3, 26, 24, SS_ICON
-    LTEXT "Informațiile de mai jos vor fi utilizate pentru identificarea calculatorului în rețea.", IDC_STATIC, 40, 5, 204, 20
-    LTEXT "Descrierea calculatorului:", IDC_STATIC, 7, 40, 90, 9
+    LTEXT "Pentru identificarea computerului în reţea, ReactOS utilizează următoarele informaţii.", IDC_STATIC, 40, 5, 204, 20
+    LTEXT "&Descriere computer:", IDC_STATIC, 7, 40, 90, 9
     EDITTEXT IDC_COMPDESC, 98, 38, 146, 13, WS_CHILD | WS_VISIBLE | WS_TABSTOP
-    LTEXT "Numele complet:", IDC_STATIC, 7, 68, 85, 9
+    LTEXT "Numele complet al computerului:", IDC_STATIC, 7, 68, 85, 9
     EDITTEXT IDC_COMPUTERNAME, 98, 68, 144, 12, ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP
-    LTEXT "Grup de lucru:", IDC_WORKGROUPDOMAIN, 7, 84, 64, 9
+    LTEXT "Domeniu:", IDC_WORKGROUPDOMAIN, 7, 84, 64, 9
     EDITTEXT IDC_WORKGROUPDOMAIN_NAME, 98, 84, 144, 12, ES_READONLY | NOT WS_BORDER | NOT WS_TABSTOP
-    LTEXT "Utilizați expertul de identificare al rețelei (apăsând „ID de rețea”) pentru a crea un utilizator și a deveni membru al unui domeniu de rețea.", IDC_STATIC, 7, 113, 172, 24
-    PUSHBUTTON  "I&D de rețea…", IDC_NETWORK_ID, 190, 114, 58, 15
-    LTEXT "Pentru a schimba identitatea calculatorului sau a intra într-un domeniu, apăsați „Schimbare”.", IDC_STATIC, 7, 149, 170, 17
-    PUSHBUTTON  "S&chimbare…", IDC_NETWORK_PROPERTY, 190, 149, 58, 15
+    LTEXT "Pentru a utiliza Expertul de identificare în reţea pentru asocierea la un domeniu şi crearea unui cont local al utilizatorului, faceţi clic pe IDentificator reţea.", IDC_STATIC, 7, 113, 172, 24
+    PUSHBUTTON  "&IDentificator reţea", IDC_NETWORK_ID, 190, 114, 58, 15
```

Comment:
looks too long to my eyes for 58 width. If it overflows, I do recommend to shorten the string.

Reply:
I checked with ReactOS's latest nightly build LiveCD (reactos-livecd-0.4.15-dev-8038-g28eb4e7-x86-gcc-lin-dbg) and it fits exactly in the box. I also checked with XP's dll and it also fits in the box. Here is the proof.

ReactOS latest nightly build
![2024-05-05 23_32_49-Resource Hacker - netid dll](https://github.com/reactos/reactos/assets/113553683/c46be3a6-9d45-4328-8ecf-6d8a9df2880d)

XP/2k3
![2024-05-05 23_33_09-Resource Hacker - netid dll mui](https://github.com/reactos/reactos/assets/113553683/311f9073-e641-479a-81b0-6be676497ec1)

Reply:
Thx. Resolved.

Reply:
No this absolutely does not convince me. The ros button is smaller than the MS button. The old master head state "&ID de reţea" does fit the button fine as we can see in your pic. But you now changed that to "&IDentificator reţea" abd that would definitely overflow the button in ros. Do not extend that strings length please.

Reply:
I understand. In this way I will keep the accelerator, but I'll revert the change.

Reply:
Ok, resolved.